test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -4303,11 +4305,41 @@ def _load_configuration(self, infile, objtype=None):
                             self.logger.error("Failed to restore "
                                               + "configuration: %s" % k)
                             return False
+                        rv = self.du.chown(path=k, runas=ROOT_USER,
+                                           uid=0, gid=0, sudo=True)
+                        if not rv:
+                            self.logger.error("Failed to restore "
+                                              + "configuration: %s" % k)
+                            return False
                     finally:
                         if os.path.isfile(fn):
-                            self.du.rm(path=fn)
+                            self.du.rm(path=fn, force=True, sudo=True)
+                return True
+            elif objtype == MGR_OBJ_NODE:
+                conf = sconf[str(objtype)]
+                nconf = {}
+                nconf = conf[str(self.hostname)]
+                for k, v in nconf.items():
+                    try:
+                        fn = self.du.create_temp_file()
```

Comment:
If `create_temp_file()` fails due to some reason, it will go to `finally` block and may be after the loop executes, it will `return True` I guess, whereas for one of the moms, the `load_configuration` might have failed. Please verify this behaviour.

Reply:
Thanks for catching it; I updated the code by removing the try block. The temp file creation for each mom happens on the local node and the file is then copied to the remote mom node.